addons/l10n_in_pos/static/src/xml/pos_receipt.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<templates id="template" inherit_id="point_of_sale.template">
+    <t t-extend="OrderReceipt">
+        <t t-jquery="div.orderlines" t-operation="before">
+            <t t-if="receipt.client">
```

Comment:
You should also check that the company is in india for when you are working in multicompany in different countries. So you can add in the 'export_for_printing' in point of sale module, the country on the company and check here in the t-if that country code is 'IN' before adding informations on the receipt.